profiles/SubhadeepZilong.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/63146468/170739449-df9a415d-7b8a-41cc-9891-5ac2d26cce86.png",
+    "issueId": 1132,
+    "name": "Subhadeep Chakraborty",
+    "username": "SubhadeepZilong"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```